proposed/2022/dotnet-nuget-config-spec.md

```diff
@@ -0,0 +1,74 @@
+# NuGet configuration CLI for dotnet.exe
+
+- Author: [Heng Liu](https://github.com/heng-liu)
+- GitHub Issue [8420](https://github.com/NuGet/Home/issues/8420)
+
+## Problem Background
+
+Currently, there is no NuGet configuration CLI for dotnet.exe. It's inconvenient for NuGet users to know about the NuGet configuration file locations and figure out where is the merged configuration coming from. 
+
+## Who are the customers
+
+This feature is for dotnet.exe users.
+
+## Goals
+Design and implement `dotnet nuget config` command.
+
+## Non-Goals
+Design and implement `dotnet nuget config` command with commands other than `list`, E.g. add/update/delete
+
+## Solution
+The following command will be implemented in the `dotnet.exe` CLI.
+
+### `dotnet nuget config`
+
+#### Commands
+If no command is specified, the command will default to list.
```

Comment:
What happens today with commands such as sources and packages? 
Is there a default command?

Reply:
Good question! There is no default command for sources and packages, not sure if it's because verbs need to go first.
But there is a default command `list` for [dotnet nuget trust command](https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-nuget-trust#commands).
So, when no sub command is specified, do we want to let people know all the subcommands for `dotnet nuget config`? Or just run dotnet nuget config list command by default?

Reply:
Hi @JonDouglas, may I know if you have any suggestions about this? Thanks! :)

Reply:
Personally, I think we should align to dotnet list package.

Reply:
Changed to no default command. Thanks!